src/components/no-options/no-options.css

```
@@ -0,0 +1,15 @@
+@import '../../styles/typography-v2.css';
+@import '../../styles/spacing.css';
+@import '../../styles/colors-v2.css';
+
+.no-options {
+  border: 1px solid var(--mds-grey-38);
+  border-radius: 3px;
+  color: var(--mds-grey-54);
+  display: flex;
+  font: var(--mds-type-content);
+  height: var(--spacing-x-large);
+  justify-content: center;
+  line-height: var(--spacing-x-large); /* stylelint-disable-line mds/typography */
```

```suggestion
  align-items: center;
```

We can use this flexbox attribute to do the same thing without needing an exception for `line-height`, which has other behavioral implications we don't want. 😄